analysisstore/client/commands.py

```diff
@@ -10,10 +10,9 @@
 
 
 class AnalysisClient:
-    """Your path to AnalysisStore(yiiiiissssss, camelcase"""
+    """Client used to pass messages between analysisstore server and apps"""
     def __init__(self, host='localhost:8999'):
```

Comment:
I have them separated in all other services. I was hoping we can move towards Olog-like port agnostic address model. I can reverse this for the time being.